.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - uses: actions/setup-python@v3
+    - uses: actions/setup-python@v4
```

Comment:
```suggestion
    - uses: actions/setup-python@v4
      with:
        python-version: "3.x"
```

Reply:
These have to be updated, see https://github.com/actions/setup-python/issues/421.